docs/how-to/contribute-code.rst

```diff
@@ -34,16 +34,26 @@ to compile this library:
 
 This will produce ``tests/objc/librubiconharness.dylib``.
 
-You can then run the test suite:
+You can then run the full test suite:
 
 .. code-block:: sh
 
     (venv) $ tox
 
-By default this will run the test suite multiple times, once on each Python version supported by Rubicon. This can take a while, so if you want to speed up the process while developing, you can run the tests on one Python version only:
+By default this will run the test suite multiple times, once on each Python
+version supported by Rubicon, as well as running some pre-commit checks of
+code style and validity. This can take a while, so if you want to speed up
+the process while developing, you can run the tests on one Python version only:
 
 .. code-block:: sh
 
     (venv) $ tox -e py37
 
+Or, to run using the version of Python that is in your current virtual
+environment:
+
+.. code-block:: sh
+
+    (venv) $ tox -e py
```

Comment:
I think we should recommend `tox -e py` first and `tox -e py37` after that. Most of the time you don't care about the exact Python version used to run the tests, you just want them to be run only once instead of on all Python versions.